### PR TITLE
Player thumbnail selection added to lobby. More changes to come.

### PR DESCRIPTION
**Pull request by AleksandrHovhannisyan**
First part of the lobby beautification is complete. The player will now see a thumbnail corresponding to the class they choose in the lobby to give them a preview.

Will work on adding a nice background and maybe some fonts.

### CODE REVIEW

File: screens/lobby_screen/lobby_screen.gd [hunk 1]
```diff
@@ -5,42 +5,53 @@ var check = true # set to false to not check for players connected
...
 func update_list():
 	connected_players.clear()
 	var players = global_player.player_info
 	for p in players:
 		connected_players.add_text(players[p]["username"] + " -> " + players[p]["classtype"] + "\n")
-	if len(players) > 1 or !check:
-		get_node("Button").disabled = false
-	else:
-		get_node("Button").disabled = true
+	$Button.disabled = (len(players) < 1 and check)
```
Comment (RustyBamboo) [inline screens/lobby_screen/lobby_screen.gd]: <= 1

**Comment by RustyBamboo** [comment]
I'd also look into how to center releative to the window size

**Comment by RustyBamboo** [comment]
And just btw, you can do git commit --amend to add current changes to previous commit (like to fix the true and <=)

File: screens/lobby_screen/lobby_screen.gd (hunk 1 above)
Comment (AleksandrHovhannisyan) [reply screens/lobby_screen/lobby_screen.gd]: I set it to < 1 to ensure that a single person can log in and play by themselves (which I think would be reasonable for the final game).

File: screens/login_screen/login_screen.gd [hunk 2]
```diff
@@ -2,7 +2,7 @@ extends Control
 
 export (PackedScene) var next_scene
 
-var check = true # set to false to disable connecting to server
+var check = false # set to false to disable connecting to server
```
[the remark this answers is not in the document]
Comment (AleksandrHovhannisyan) [reply screens/login_screen/login_screen.gd]: Alright, I'll go ahead and change that

**Comment by AleksandrHovhannisyan** [comment]
Yep, I'll look into centering as well

File: screens/lobby_screen/lobby_screen.gd (hunk 1 above)
Comment (RustyBamboo) [reply screens/lobby_screen/lobby_screen.gd]: aight

**Pull request merged**